src/Fable.React.Props.fs

```diff
@@ -872,6 +1087,11 @@ type CSSProp =
     | Zoom of obj
     /// If you are searching for a way to provide a value not supported by this DSL then use something like: CSSProp.Custom ("align-content", "center")
     | [<Erase>] Custom of string * obj
+    static member Overflow (overflow: OverflowOptions, ?overflowY: OverflowOptions) =
+        match overflowY with
+        | Some value -> CSSProp.Custom ("overflow",  unbox overflow + " " + unbox value)
+        | None -> CSSProp.Custom ("overflow",  unbox overflow)
```

Comment:
Hmm, not sure if this will work on .NET side for SSR, but this makes me realize we may also have a problem with other StringEnum options. I need to look into that.